core/modules/entity/tests/entity_test/entity_test.entity.inc

```diff
@@ -1,4 +1,5 @@
 <?php
+
```

Comment:
I think we've actually been removing these empty lines at the top of files (differing from Drupal).

Reply:
Was that documented? I didn't see it anywhere.

Reply:
If not, we should add it!